### DISCUSSION TITLE
Executables meshroom_status, etc.

### DISCUSSION THREAD

**Discussion by mokrueger** [alicevision/Meshroom#1409] [Q&A]
Why are the other scripts like /bin/meshroom_status not included in the executable version?
Are these options deprecated or why are they missing? 

I wanna use meshroom in my python application and would like to get status information easily.
Thanks in advance.
(Instructions to build it manually would also be appreciated)

**Comment by natowi** [comment]
Resolved by https://github.com/alicevision/meshroom/pull/1419
